NomaiVR/manifest.json

```diff
@@ -4,7 +4,7 @@
   "name": "NomaiVR",
   "uniqueName": "Raicuparta.NomaiVR",
   "description": "Outer Wilds VR mod, with full motion control support.",
-  "version": "1.0.2",
-  "owmlVersion": "1.0.1",
+  "version": "1.0.3",
```

Comment:
Please revert this change, NomaiVR's version will be bumped in a dedicated commit. Also, this is a new feature, so we'll bump the minor instead of patch version (1.1.0).